### DISCUSSION TITLE
Compilation Instructions for Production Hubs and Reticulum

### DISCUSSION THREAD

**Discussion by NagarajuSamudrala** [mozilla/hubs#5021] [General Questions & Discussion]
Hello,

I am able to compile Reticulum and Hub for Local.
Please let me know what are the **change in the instructions for production** ( This is for any server, not only AWS) server ..

I used instructions from following to compile for Local ..
https://github.com/mozilla/reticulum
https://github.com/mozilla/hubs

Thanks,
Nagaraju Samudrala

**Comment by pattersonbl2** [comment]
This question would be great to also ask in our community discord. You can join here (https://discord.com/invite/dFJncWwHun). I will ask my team if they may have a quick answer as well.
